_posts/samus_returns/2024-06-11-export-exheader.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: "How to extract your Metroid: Samus Returns ExHeader"
+tags: samus_returns
+---
+You can either extract your ExHeader using Godmode9 from your 3DS, or using Citra from an installation file (.3ds or .cia) and then using [ctrtool](https://github.com/3DSGuy/Project_CTR/releases/tag/ctrtool-v1.2.0) or [pyctr](https://github.com/ihaveamac/pyctr). We'll show you how to do it using Godmode9.
```

Comment:
The text is explicitly about extracting the exheader.bin. Why does it mention Citra? If you have a 3ds file, there is nothing you can do with Citra to extract the exheader.bin. Especially because the text is naming different tools.
Imo it should be more like "use Godmode9 on 3DS or blablabla on PC"

Reply:
Honestly I shouldn't even list anything else other than godmode9 since a modded 3ds is expected and that is the best way to do everything. You wouldn't not have that installed.